docker-compose.yml

```diff
@@ -354,7 +354,7 @@ services:
         ports:
             - "${POSTGRES_PORT}:5432"
         healthcheck:
-            test: ["CMD", "pg_isready", "-U", "${PIXL_DB_USER}"]
+            test: ["CMD", "pg_isready", "-U", "${PIXL_DB_USER}", "--dbname", "${PIXL_DB_NAME}"]
```

Comment:
this is unrelated, but without adding `--dbname` there were confusing errors in the logs from the healthcheck:
> postgres-1  | 2024-08-23 15:08:32.833 GMT [77] FATAL:  database "pixl_db_username" does not exist
postgres-1  | 2024-08-23 15:08:42.882 GMT [85] FATAL:  database "pixl_db_username" does not exist
postgres-1  | 2024-08-23 15:08:52.951 GMT [93] FATAL:  database "pixl_db_username" does not exist

Reply:
ooh yeah that's definitely not what you want